keyboards/keebio/quefrency/keymaps/unausgeschlafen/rules.mk

```diff
@@ -0,0 +1,2 @@
+BOOTLOADER = atmel-dfu # comment this line if you use arduinos
+DISABLE_JTAG = yes
```

Comment:
This isn't actually needed. 

If you're on a recent version of QMK, JTAG is always disabled, now. 

```suggestion
```